### ISSUE TITLE
Feature Request: Option to disable sync while on battery

### ISSUE THREAD

**Issue by scienmind** [canton7/SyncTrayzor#283]
Just as SyncTrayzor handles metered connections to save data, I think it may be a good idea to provide similar mechanism to save battery life.

While working on laptops (on battery) and frequently saving small edits of files, ST becomes a major battery killer (probably rightfully so due to encryption). It would usually cut battery time to half.

So it may be nice to add "power source awareness" to the wrapper do be able to (optionally) disable sync while on battery, and re-enable it when back on AC.

**Comment by canton7** [comment]
Yep, fair enough request.

**Comment by benshep** [comment]
Please consider this a +1 from me.

**Comment by Hard-Wired** [comment]
Additionally, it would be nice to have an option to automatically Pause if the system is in Battery Saver mode.

**Comment by hyperloop512** [comment]
Yes please. Or an option to right-click on the tray icon and pause would also work.
And thanks for this app...
